examples/.gitignore

```diff
@@ -0,0 +1,4 @@
+!factors/Cargo.lock
+!factors/methods/guest/Cargo.lock
```

Comment:
This is fine, because it's actually an isolated workspace (see the `[workspace]` section, that's what makes it think its a workspace).